src/jsdom.js

```diff
@@ -1,22 +1,9 @@
 'use strict';
 
 const jsdom = require('jsdom/lib/old-api.js'),
-    Promise = require('bluebird'),
     path = require('path'),
     _ = require('lodash');
 
-// Configure.
-const jsdomAsync = Promise.promisify(jsdom.env, { context: jsdom });
-
-/**
- * Closes a page.
- * @param {Object} Page opened by jsdom
- * @return {void}
- */
-function cleanup(page) {
-    return page.close();
```

Comment:
Where does this line go, is anything cleaning up our jsdom pages in the new codebase? Or if not, why is it no longer needed?